src/oci_cli/cli_setup_bootstrap.py

```diff
@@ -114,8 +134,12 @@ def create_user_session(region='', tenancy_name=None):
     if region == '':
         region = cli_setup.prompt_for_region()
 
+    
+
     # try to set up http server so we can fail early if the required port is in use
     try:
+        # Firstly, we will check if PORT is available or not
+        BOOTSTRAP_SERVICE_PORT=is_port_available(BOOTSTRAP_SERVICE_PORT)
```

Comment:
Convention requires constants to be referenced in uppercase and immutable, so I would suggest creating `DEFAULT_BOOTSTRAP_SERVICE_PORT` as `8181` (at the top of the file) and then setting `boot_strap_service_port` here in lowercase, as it's a variable, not a constant.

Also, shouldn't this be calling `find_port` instead of `is_port_available`?

Reply:
yes, it should be calling find_port, just saw that typo.